framework/base/variables/_font-family.scss

```diff
@@ -1,5 +1,5 @@
 // ECL Font family
 
-$font-family-sans-serif: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue', Arial, sans-serif !default;
+$font-family-sans-serif: 'Arial', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue', Arial, sans-serif !default;
```

Comment:
`'Arial'` is already in the stack. As discussed with @degliwe yesterday, I'd just keep the stack as it is right now (silent enhancement)